code/src/components/Survey.js

```diff
@@ -0,0 +1,207 @@
+  const [checkboxGroup, setCheckboxGroup] = useState([])
+  const [showSummary, setShowSummary] = useState(false)
+
+  //VARIABLES
+
+  const enjoyReadingArray = ['I enjoy reading a lot', 'I like reading sometimes', 'I do not like reading']
+  const checkboxGroupArray = ['Adventure', 'Classics', 'Detective', 'Fantasy', 'History', 'Romance', 'Science Fiction', 'Poetry', 'Self-Help', 'Other']
+
+  //FUNCTIONS
+
+  // This function leads to the next question
+  const onNextQuestionChange = () => {
+    setQuestion(question + 1)
+  }
+
+  // This function goes back to the previous question
+  const onPreviousQuestionChange = () => setQuestion(question - 1)
+
+  //This function will show the summary of answers and prevet the page from refreshing on submit form
+  const handleSubmit = event => {
+    event.preventDefault()
+    setShowSummary(true)
+  }
+  
+  // This function stores the answer from the first question (radio input)
+  const onEnjoyReadingChange = event => setEnjoyReading(event.target.value)
+
+  // This function stores the answer from the second question (select input)
+  const onReadingFormatChange = event => setReadingFormat(event.target.value)
+
+  // This function stores the answer from the third question (text input)
+  const onBookChange = event => setBook(event.target.value)
+
+  // This function stores the answer from the fourth question (checkbox input)
+  const onIsCheckedChange = event => setIsChecked(event.target.checked)
+
+  // This function will update the value of an array/answer for the fifth question (checkbox group input)
+  const onCheckboxGroupToggle =(checkboxValue) => {
+    if (checkboxGroup.includes(checkboxValue)) {
+      // delete element from array by using filter method
+      setCheckboxGroup(checkboxGroup.filter(item => item !== checkboxValue))
+    } else {
+      // add element to array. Spread operator (...) will copy all the values from the previous heckboxGroup array 
+      setCheckboxGroup([checkboxValue, ...checkboxGroup])
+    }
+  }
+
+
+  //RENDERING
+
+  return (
+    <main className='main-container'>
+      {/*This will show the the questions if the form is not submitted*/}
+      {!showSummary ? (
+      <form className='form' onSubmit={handleSubmit}>
+
+        {/* Survey starts here  */}
+        {question === 0 && (
+          <section className='start-page'>
+            <h1 className='header'>READING HABITS SURVEY</h1>
+            <p className='small-text'>Time to complete: 3 minutes</p>
+            <div className="start-button-container">
+              <Button onChangeDirection={onNextQuestionChange} textDisplay={'Start the survey'} className={'start-button'} />
+            </div>
+          </section>
+        )}
+
+        {/* First question */}
+        {question === 1 && (
+          <section className='question-container'>
+            <h2 tabindex='0' className='question'>How much do you enjoy reading?</h2>
+            <div className="radio-buttons">
```

Comment:
You sometimes have the question as an h2 in the survey.js and sometimes in the separate component. Does this have a reason or could it be for consistency for all the same? 🧐